### DISCUSSION TITLE
How do you remove vanjaro from dnn?

### DISCUSSION THREAD

**Discussion by mastercodefu** [vanjarosoftware/Vanjaro.Platform#2333] [Q&A]
Any ideas? Or am i missing any guides.

MK

**Comment by meetmandeep** [comment]
Uninstall the extension like you would any other DNN Extension; be sure to remove any reference to Vanjaro from web.config

**Comment by mastercodefu** [comment]
Thanks Meetmandeep.  I love the project minor frustrations with dnn integration.  I appreciate though is a really great project for none web designers and more admins.  Will follow and looking forward to updates.

Craig
